packages/react-params/src/initValue.ts

```diff
@@ -89,7 +89,7 @@ export default function getInitValue (def: TypeDef): unknown {
       return createType(registry, 'H512');
 
     case 'H160':
-        return createType(registry, 'H160');
+      return createType(registry, 'H160');
```

Comment:
```suggestion
      return registry.createType('H160');
```

(I have the others changed in a different PR, for some reason the `createType(registry, ...` form is still in some places in apps)

Reply:
ty